src/components/__tests__/Weather.test.tsx

```diff
@@ -0,0 +1,63 @@
+import axios from 'axios';
+import { fetchWeatherData } from '../../__mocks__/WeatherMocks';
+import { render, waitFor } from '@testing-library/react';
+import Weather from '../Weather';
+
+jest.mock('axios');
+const mockedAxios = axios as jest.Mocked<typeof axios>;
+
+describe('mock api calls', () => {
+  beforeEach(() => {
+    mockedAxios.get.mockResolvedValue({
+      data: {
+        result: {
+          weather: {
+            forcast: 'Sunny',
+            max: 28,
+            min: 17,
+            description: 'Clear skys all day with a warm summber breaze ariving in the afternoon',
+          },
+        },
+      },
+    });
+  });
+
+  test('should return description', async () => {
+    const { getByText, getByTestId } = render(<Weather />);
+    await waitFor(() => {
+      fetchWeatherData();
```

Comment:
No need to call `fetchWeatherData` as the API is called inside the Weather component